commerce-media-web/build.gradle

```diff
@@ -1,11 +1,28 @@
 apply plugin: "com.liferay.lang.merger"
 
 dependencies {
-	compileOnly group: "com.liferay.commerce", name: "com.liferay.commerce.media.api", version: "1.0.0"
-	compileOnly group: "com.liferay.commerce", name: "com.liferay.commerce.product.api", version: "3.0.0"
+	compileOnly group: "biz.aQute.bnd", name: "biz.aQute.bndlib", version: "3.1.0"
+	compileOnly group: "com.liferay", name: "com.liferay.frontend.taglib", version: "3.0.0"
+	compileOnly group: "com.liferay", name: "com.liferay.item.selector.api", version: "3.0.0"
+	compileOnly group: "com.liferay", name: "com.liferay.item.selector.criteria.api", version: "3.0.0"
+	compileOnly group: "com.liferay", name: "com.liferay.item.selector.taglib", version: "2.0.0"
+	compileOnly group: "com.liferay", name: "com.liferay.osgi.service.tracker.collections", version: "3.0.0"
+	compileOnly group: "com.liferay", name: "com.liferay.petra.lang", version: "2.0.0"
+	compileOnly group: "com.liferay", name: "com.liferay.petra.string", version: "2.0.0"
+	compileOnly group: "com.liferay", name: "com.liferay.portal.configuration.metatype.api", version: "1.0.0"
+	compileOnly group: "com.liferay", name: "com.liferay.upload.api", version: "2.0.0"
 	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "3.5.0"
+	compileOnly group: "com.liferay.portal", name: "com.liferay.util.taglib", version: "3.0.0"
 	compileOnly group: "javax.portlet", name: "portlet-api", version: "3.0.0"
 	compileOnly group: "javax.servlet", name: "javax.servlet-api", version: "3.0.1"
+	compileOnly group: "org.osgi", name: "org.osgi.core", version: "5.0.0"
+	compileOnly group: "org.osgi", name: "org.osgi.service.cm", version: "1.5.0"
 	compileOnly group: "org.osgi", name: "org.osgi.service.component.annotations", version: "1.3.0"
+	compileOnly project(":apps:commerce:commerce-admin-api")
 	compileOnly project(":apps:commerce:commerce-api")
+	compileOnly project(":apps:commerce:commerce-media-api")
+	compileOnly project(":apps:commerce:commerce-product-api")
+	compileOnly project(":apps:commerce:commerce-product-definitions-web")
```

Comment:
I don't think for this module we do need all of this dependencies listed, form portal and from commerce. Can we clean ?